index.html

```diff
@@ -77,6 +80,16 @@ <h2 class="Footer__title">Made with all the love we could be bothered to muster<
 			function gtag(){dataLayer.push(arguments);}
 			gtag('js', new Date());
 			gtag('config', 'UA-39417124-2');
-		</script>	
+		</script>
+		
+		<script>
+			// Add Service Worker for the app to work offline
+			window.addEventListener("load", async _ => {
+				if('serviceWorker' in navigator) {
+					await navigator.serviceWorker.register('/sw.js');
+					console.log('[Service Worker] Registered');
+				};
+			});
```

Comment:
Can arrow functions and async/await be avoided at this point? (they're OK inside `sw.js`)

This repo doesn't get processed with Babel at any point, so this will causes errors in older browsers that don't support the syntax.